components/centraldashboard/app/server.ts

```diff
@@ -41,7 +45,11 @@ async function main() {
 
   const app: express.Application = express();
   const k8sService = new KubernetesService(new KubeConfig());
-  const metricsService = await getMetricsService(k8sService);
+
+  const metricsService = PROMETHEUS_URL
```

Comment:
I think we will have to make sure that this works with our prometheus instance if we want to get our correct metrics.

Reply:
Comment added to https://jirab.statcan.ca/browse/BTIS-623